### DISCUSSION TITLE
Error with the orbit files from ASF

### DISCUSSION THREAD

**Discussion by alexisInSAR** [alexisInSAR/EZ-InSAR#82] [Ideas and further developments]
Hi all, 

A quick fix was done for downloading orbit files from the ASF server. Unfortunately, the webpage of RES orbit files is no longer available due to the number of file. However the POE file can be downloaded without issue. 

I have decided to change the server to [Copernicus data space server](https://dataspace.copernicus.eu). Please do not forget to have an account. 

The next version will be available as soon as possible. 

Kind regards, 

Alexis

**Comment by alexisInSAR** [comment]
Hi all, 

The new version is available (2.2.2 Beta). 

Kind regards, 

Alexis

**Reply by geoenviron98** [reply]
Hi Alex

I tried to download the Orbit files from 2.2.3 version, but after entering the Username (email) and password, I only got this progress. Do you know what was happen here? 

![WhatsApp Image 2024-05-13 at 10 20 34](https://github.com/alexisInSAR/EZ-InSAR/assets/169086903/d7017933-6366-405d-a8d3-f54bf4bc9408)

**Reply by geoenviron98** [reply]
I have solved that problem. The problem appear because I used ASF account instead of Copernicus account
